acapy_plugin_toolbox/invitations.py

```diff
@@ -0,0 +1,186 @@
+"""Define messages for connections admin protocol."""
+
+# pylint: disable=invalid-name
+# pylint: disable=too-few-public-methods
+
+from marshmallow import fields, validate
+
+from aries_cloudagent.config.injection_context import InjectionContext
+from aries_cloudagent.core.protocol_registry import ProtocolRegistry
+from aries_cloudagent.messaging.base_handler import BaseHandler, BaseResponder, RequestContext
+from aries_cloudagent.protocols.connections.manager import ConnectionManager
+from aries_cloudagent.connections.models.connection_record import (
+    ConnectionRecord, ConnectionRecordSchema
+)
+from aries_cloudagent.protocols.connections.messages.connection_invitation import (
+    ConnectionInvitation,
+)
+from aries_cloudagent.protocols.problem_report.message import ProblemReport
+from aries_cloudagent.storage.error import StorageNotFoundError
+from aries_cloudagent.messaging.valid import INDY_ISO8601_DATETIME
+
+from .util import generate_model_schema, admin_only
+
+PROTOCOL = 'https://github.com/hyperledger/aries-toolbox/tree/master/docs/admin-invitations/0.1'
+
+# Message Types
+INVITATION_GET_LIST = '{}/invitation-get-list'.format(PROTOCOL)
+INVITATION_LIST = '{}/invitation-list'.format(PROTOCOL)
+CREATE_INVITATION = '{}/create-invitation'.format(PROTOCOL)
+INVITATION = '{}/invitation'.format(PROTOCOL)
```

Comment:
Since this protocol is limited to only invitations (whereas connections and invitations were originally combined), maybe we can drop invitation from the message names?
```suggestion
INVITATION_GET_LIST = '{}/get-list'.format(PROTOCOL)
INVITATION_LIST = '{}/list'.format(PROTOCOL)
CREATE_INVITATION = '{}/create'.format(PROTOCOL)
INVITATION = '{}/invitation'.format(PROTOCOL)
```